data.sql

```diff
@@ -6,9 +6,30 @@ INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALU
 
 --PROJECT TWO
 INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Charmander','2020-02-08',false,-11,0);
-INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Plantmon','2021-11-15',true,-5.7,2);
+INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Plantmon',);
 INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Squirtle','1993-04-02',false,-12.13,3);
 INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Angemon','2005-06-12',true,-45,1);
 INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Boarmon','2018-06-07',true,20.4,7);
 INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Blossom','1998-10-13',false,17,3);
 INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Ditto','2022-05-14',true,22,4);
+
+--PROJECT THREE
+INSERT INTO owners (fullname,age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (fullname,age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners (fullname,age) VALUES ('Bob', 45);
+INSERT INTO owners (fullname,age) VALUES ('Melody Pond', 77);
+INSERT INTO owners (fullname,age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (fullname,age) VALUES ('Jodie Whittaker', 38);
+
+INSERT INTO species (name,age) VALUES ('Pokemon');
+INSERT INTO species (name,age) VALUES ('Digimon');
+
```

Comment:
- [ ] When you created the `species` table, there is no column for `age`, but now that you are trying to insert data into the table, you are declaring the `age` column. Kindly remove the `age` column because it is not in the requirements. 👍